thoth/messaging/message_base.py

```diff
@@ -58,6 +61,17 @@ def __init__(
         self.topic_retention_time_second = topic_retention_time_second
         self.protocol = os.getenv("KAFKA_PROTOCOL") or protocol
 
+        if MessageBase.app is None:
```

Comment:
Isn't this going to start app many times when we use more messages?

Reply:
No, because this is the `superclass` and `MessageBase.app` is a shared variable rather than an `attribute` it shouldn't.

Reply:
Thanks for explaining! 🏅 💯

Reply:
For proof rather than just my word
```
Python 3.8.3 (default, May 29 2020, 00:00:00) 
[GCC 10.1.1 20200507 (Red Hat 10.1.1-1)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> class A:
...     doc = None
...     def __init__(self):
...             if A.doc is None:
...                     A.doc = True
...                     print("hello")
... 
>>> a = A()
hello
>>> b = A()
>>> 
```

Reply:
Thanks, very nice! And I also trust your word 🥇